modules/apps/commerce/commerce-service/src/main/resources/com/liferay/commerce/internal/batch/returns.batch-engine-data.json

```diff
@@ -0,0 +1,126 @@
+{
+	"configuration": {
+		"className": "com.liferay.object.admin.rest.dto.v1_0.ObjectDefinition",
+		"multiCompany": true,
+		"parameters": {
+			"containsHeaders": "true",
+			"createStrategy": "UPSERT",
+			"featureFlag": "LPD-9361",
+			"importStrategy": "ON_ERROR_FAIL",
+			"taskItemDelegateName": "DEFAULT",
+			"updateStrategy": "UPDATE"
+		},
+		"userId": 0,
+		"version": "v1.0"
+	},
+	"items": [
+		{
+			"enableCategorization": true,
+			"externalReferenceCode": "L_RETURN",
+			"label": {
+				"en_US": "Return"
+			},
+			"modifiable": true,
+			"name": "Return",
+			"objectFields": [
+				{
+					"DBType": "String",
+					"businessType": "Picklist",
+					"externalReferenceCode": "RETURN-STATE",
+					"indexed": true,
+					"indexedAsKeyword": false,
+					"indexedLanguageId": "en_US",
+					"label": {
+						"en_US": "ReturnState"
+					},
+					"listTypeDefinitionExternalReferenceCode": "RETURN-STATE_PICKLIST",
```

Comment:
Hey @CrescenzoRegaLF it is better to prefix all the ERCs with `L_` not only the ones for object definitions (Brian told us that in a different PR).